docs/customer-portal/cp-dashboard.md

```diff
@@ -41,3 +57,17 @@ You can perform the following Functions with this data:
      For billing purposes, consider the data found in the Customer Summary section as valid. Although the same information might be at other locations, you shouldn't use it for billing.
 
 [cp-dash]: /customer-portal/img/cp-dash.png "Top of Page"
+
+## Internal Number Block
+
+Internal Number Block to define the range of numbers a Customer can use for setting up SIP Extensions.
+
+### Why Define Internal Number Blocks?
+
+Within each Account are multiple Customers, each will likely have more than one Contact.

```

Comment:
No need for extra information on Internal Number Block as its Customer Portal documentation 
Just define what is internal Number Block and they can see what range has been assigned to them

Reply:
removed extra information